azurerm/internal/services/healthcare/healthcare_service_data_source.go

```diff
@@ -42,6 +42,11 @@ func dataSourceHealthcareService() *schema.Resource {
 				Computed: true,
 			},
 
+			"cosmosdb_key_vault_key_id": {
```

Comment:
could we include that it needs to be versionless here?
```suggestion
			"cosmosdb_key_vault_key_versionless_id": {
```

Reply:
I was inspired by the cosmosdb attribute [here](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/cosmosdb_account#key_vault_key_id). If you don't mind this inconsistency I'm fine with the suggestion.

Reply:
Do you still think I should make this change @katbyte?

Reply:
Not at all, we're going to move toward including versionless going forward to indicate what type of key to use

Reply:
I updated the name of this attribute.